src/docs/components/docs-nav/docs-nav.css

```diff
@@ -195,20 +210,7 @@
 
   a {
     padding-left: var(--ld-sp-40);
-
-    &::after {
-      position: absolute;
-      right: 0;
-      top: calc(-1 * var(--ld-sp-1));
-      background-repeat: no-repeat;
-      background-position: center;
-      width: 0.8rem;
-      height: 1rem;
-      display: inline-flex;
-      align-items: center;
-      justify-content: center;
-      pointer-events: none;
-    }
+    pointer-events: all;
```

Comment:
This applies to svg only, according to MDN. Does this work with inheritance?

Reply:
Yes it works well. Have been using this technique for years 🤓